python/examples/tools/advanced.py

```diff
@@ -0,0 +1,17 @@
+import asyncio
+
+from beeai_framework.tools.weather.openmeteo import OpenMeteoTool, OpenMeteoToolInput
+
+
+async def main() -> None:
+    tool = OpenMeteoTool()
```

Comment:
compared to TS you skipped the options input here and removed the following logging, is there a reason?